koku/masu/util/oci/common.py

```diff
@@ -161,3 +161,15 @@ def detect_type(report_path):
     else:
         report_type = "usage"
     return report_type
+
+
+def deduplicate_reports_for_oci(report_list):
+    """Remove duplicate oci manifests"""
+    reports_deduplicated = []
+    manifest_id_set = set()
+    for report in report_list:
+        _manifest_id = report.get("manifest_id")
+        if _manifest_id and _manifest_id not in manifest_id_set:
```

Comment:
I was curious if the walrus operator would work here, so I tried it.
```suggestion
        if (_manifest_id := report.get("manifest_id")) and _manifest_id not in manifest_id_set:
```

I don't think this is better (more readable) than what you have currently, but it's interesting.

Reply:
I'm going to lean towards readability here 😀